artifacts/camara-cloudevents/subscription-notification-template.yaml

```
@@ -0,0 +1,759 @@
+openapi: 3.0.3
+info:
+  title: Notification Subscription Template
+  description: |
+    This file is a template for CAMARA explicit subscription template and for Notification model. Additional information are provided in API Design guidelines document.
```

duplication of `template` word - it can be removed or changed like this:
```suggestion
    This file is a template for CAMARA API explicit subscription endpoint and for Notification model. Additional information are provided in API Design Guidelines document.
```